src/Google/GlobalSiteTag.php

```diff
@@ -344,6 +440,20 @@ private function display_page_view_event_snippet(): void {
 		wp_print_inline_script_tag( $page_view_gtag );
 	}
 
+	/**
+	 * Add product data to include in JS data.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param WC_Product $product
+	 */
+	protected function add_product_data( $product ) {
+		$this->products[ $product->get_id() ] = [
```

Comment:
💅  Not sure how strict we are here about using shorthands [] instead of array()

Reply:
For this extension we have always encouraged it, and most of the code is written using shorthand arrays.

See: https://github.com/woocommerce/google-listings-and-ads/blob/1.13.2/phpcs.xml.dist#L31-L32